src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java

```diff
@@ -92,6 +97,27 @@ public BranchApi branch(String ref) throws RestApiException {
         JsonElement branches = gerritRestClient.getRequest(request);
         return branchInfoParser.parseBranchInfos(branches);
     }
+    
+    @Override
+    public ListRefsRequest<TagInfo> tags() {
+        return new ListRefsRequest<TagInfo>() {
+            @Override
+            public List<TagInfo> get() throws RestApiException {
+                return ProjectApiRestClient.this.getTagInfos(this);
+            }
+        };
+    }
+    
+    @Override
+    public TagApi tag(String ref) throws RestApiException {
+        return new TagApiRestClient(gerritRestClient, tagInfoParser, this, ref);
+    }
+    
+    private List<TagInfo> getTagInfos(ListRefsRequest<TagInfo> lbr) throws RestApiException {
```

Comment:
Are you sure?

At least in Gerrit REST API documentation of the latest Gerrit version, options limit and skip are documented. See: https://gerrit-review.googlesource.com/Documentation/rest-api-projects.html#list-tags

I think we should implement the available options and throw a NotImplementedException for requested options which are not supported. Otherwise it just silently fails.